cfgmgr/buffer_headroom_mellanox.lua

```diff
@@ -39,6 +39,7 @@ local ret = {}
 -- the key of table pause_quanta_per_speed is operating speed at Mb/s
 -- the value of table pause_quanta_per_speed is the number of pause_quanta
 local pause_quanta_per_speed = {}
+pause_quanta_per_speed[800000] = 905
```

Comment:
how is this the same as for 400G?

Reply:
Yes, this is defined by the IEEE. 400G and 800G share the same value.